### ISSUE TITLE
using with wordpress

### ISSUE THREAD

**Issue by designlobby** [nicinabox/superslides#138]
I'm trying to set this up on a local wordpress install. Everything works except the images don't resize when the browser window resizes, they stay at the size they were when the page first loads. What would stop the images resizing?

**Comment by nicinabox** [comment]
Maybe you have a js error?

**Comment by designlobby** [comment]
Yeah that's what I though, but there are no errors when I look in Firebug. Everything else seems fine, slides are working, pagination, speeds etc.

**Comment by nicinabox** [comment]
I think I would need to see it to tell you more. You haven't put `.preserve` on the background images?

**Comment by designlobby** [comment]
No, I haven't added any classes to the images. I can upload what I have to a test domain tomorrow and post the url here. Thanks for your help Nic.

**Comment by michaelryanmcneill** [comment]
The issue is associated with the fact that WordPress jQuery is loaded in noConflict mode. To fix this, put your options inside of `jQuery(function ($) {  });`, as shown below:

```
jQuery(function ($) {
    $('#slides').superslides({
        hashchange: true,
        play: 2000
      });
}); 
```

**Comment by jakesteele** [comment]
I also had this problem, the images would leave a small border looking area on the side or bottom. To fix this I set a timeout after the .start(); call on ln 98 (10 ms) and that fixed my problem.
